bitex/api/REST/coinone.py

```diff
@@ -0,0 +1,53 @@
+"""CoinOne REST API backend.
+
+Documentation available at:
+    https://coinone.co.kr/developer/
+"""
+# pylint: disable=too-many-arguments
+# Import Built-ins
+import logging
+import json
+import hashlib
+import hmac
+import base64
+
+# Import Third-Party
+
+# Import Homebrew
+from bitex.api.REST import RESTAPI
+
+log = logging.getLogger(__name__)
+
+
+class CoinoneREST(RESTAPI):
+    """BitHumb REST API class."""
+
```

Comment:
C&P leftover here as well

Reply:
Ha-ha, same mistake in your BinanceREST.
Today, binance wallet can't work, 403 Forbidden, I didn't found the reason. No code change.

Reply:
modified